app/views/users/_form.html.erb

```diff
@@ -20,8 +20,8 @@
 
   <div class='tab-content'>
     <div class='tab-pane active' id='primary'>
-
-      <%= text_f f, :login, :disabled => f.object.login == 'admin' %>
+      <% can_edit_login =   User.current.can?(:edit_users, @user) || ( f.object.id == User.current.id  && User.current.internal? ) %>
+      <%= text_f f, :login, :disabled => !can_edit_login, :label_help => !can_edit_login ? "The user can not edit the login":" " %>
```

Comment:
This ought to be translatable: `_("The user can not edit the login")`.

How about "Only internal users with enough permissions can edit their login". I'm not too sure of the wording though.. (cc @ehelms @GregSutcliffe mind to double check?)

Reply:
Hmm, tricky one. Perhaps "Only permitted local users can edit their login"? Not sure about loca vs internal - local seems more "friendly" but possible to misinterpret?

Reply:
Another idea, two tests with two messages. If it's an external user, display "External users cannot edit their login", and if it's an internal user, check perms and display "You do not have permission to edit your login" (or whatever we use elsewhere for not having the right permissions".